[
    {
        "content": "<p><a href=\"https://github.com/leanprover-community/tutorials/blob/9333ac30d79bce6d8e0e2951166f5cd9494f6508/src/exercises/05_sequence_limits.lean#L3\">this line</a> has an error:</p>\n<div class=\"codehilite\" data-code-language=\"Lean\"><pre><span></span><code><span class=\"bp\">â–¶</span> <span class=\"mi\">4</span><span class=\"o\">:</span><span class=\"mi\">1</span><span class=\"bp\">-</span><span class=\"mi\">4</span><span class=\"o\">:</span><span class=\"mi\">2</span><span class=\"o\">:</span> <span class=\"n\">error</span><span class=\"o\">:</span>\n<span class=\"n\">invalid</span> <span class=\"kn\">import</span><span class=\"o\">:</span> <span class=\"n\">tuto_lib</span>\n<span class=\"n\">could</span> <span class=\"n\">not</span> <span class=\"n\">resolve</span> <span class=\"kn\">import</span><span class=\"o\">:</span> <span class=\"n\">tuto_lib</span>\n</code></pre></div>\n<p>is it because of an updated mathlib? How should I fix it?<br>\n<span class=\"user-mention\" data-user-id=\"112680\">@Johan Commelin</span> you might be interested?</p>",
        "id": 307893691,
        "sender_full_name": "Siyuan Yan",
        "timestamp": 1667544851
    },
    {
        "content": "<p>ohh, it's a local file in the solution directory but not in the exercise directory.</p>",
        "id": 307893951,
        "sender_full_name": "Siyuan Yan",
        "timestamp": 1667545048
    }
]